users/test/test_serializers.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth import get_user_model
+from faker import Faker
+from rest_framework.test import APIClient, APITestCase
+
+from users.serializers import ProfileSerializer
+
+User = get_user_model()
+fake = Faker()
+
+
+class ProfileTest(APITestCase):
+    def setUp(self) -> None:
+        self.user = User.objects.create(
+            username=fake.name(),
+            email=fake.email(),
+            password=fake.password(),
+        )
+
+        self.client = APIClient()
+
+        self.serializer_data = {"bio": "keep at it"}
+
+    def test_serializers_has_expected_fields(self) -> None:
```

Comment:
This test does not make sense, it is not calling the serializer so how are you testing it?